packages/next-plugin-sentry/src/on-error-server.js

```diff
@@ -0,0 +1,36 @@
+import { captureException, flush, Handlers, withScope } from '@sentry/nextjs';
+import getConfig from 'next/config';
+
+const { parseRequest } = Handlers;
+
+export default async function onErrorServer(err) {
+  const { serverRuntimeConfig = {}, publicRuntimeConfig = {} } = getConfig() || {};
+  const sentryTimeout = serverRuntimeConfig.sentryTimeout || publicRuntimeConfig.sentryTimeout || 2000;
+
+  withScope(scope => {
+    if (typeof err.req !== 'undefined') {
```

Comment:
Any reason to use `typeof` instead of this:

```suggestion
    if (err.req !== undefined) {
```

?